src/app.ts

```diff
@@ -187,6 +189,21 @@ export type ContextData = {
 
 const fedifyKv = await KnexKvStore.create(client, 'key_value');
 
+logging.info('Initialising GCP storage service');
+const gcpStorageService = new GCPStorageService();
+
+try {
+    logging.info('Initialising GCP storage service');
+    await gcpStorageService.init();
+    logging.info('GCP storage service initialised');
+    logging.info('Adding latest code for GCP storage service');
```

Comment:
This is some extra logging for testing on stg, will remove once testing is complete